charts/graylog/values.yaml

```diff
@@ -342,6 +342,9 @@ graylog:
     ## If true, prometheus annotations will be attached
     ##
     enabled: false
+    serviceMonitor:
+      enabled: false
+      additionalLabels: {}
```

Comment:
Please add missing `interval` and `scrapeTimeout` in value file too. And write some short description of the purpose of them.

Reply:
done added now